### PR TITLE
fixed width and margins to center bell

### PR DESCRIPTION
**Pull request by tangerine007**
422-bell

### CODE REVIEW

File: app/assets/stylesheets/common/_layout.sass [hunk 1]
```diff
@@ -59,6 +59,10 @@ body
       line-height: 17px
       font-size: 14px
       padding: 9px 12px
+      i
```
Comment (stackus) [inline app/assets/stylesheets/common/_layout.sass]: Have you tested this with a wide and narrow browser window? These look like they're specific to a certain window width.

Comment (tangerine007) [inline app/assets/stylesheets/common/_layout.sass]: It is fixed pixel width for the column, I tried yesterday and it kept it at 64px no matter the window size. Heroku isn't launching for me atm so I can't pull it up right now.

Comment (chriskacerguis) [inline app/assets/stylesheets/common/_layout.sass]: @tangerine007 Heroku is back up, can you take a look here and let me know?

Comment (tangerine007) [inline app/assets/stylesheets/common/_layout.sass]: Heroku is down for me again... very frustrating. I am pretty sure this works fine, unless 
```text-align: center``` would be better?

**Comment by calebthompson** [comment]
:+1:

File: public/templates/partials/common/problem_alerts.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
             </thead>
             <tbody>
             <tr ng-repeat="row in alerts">
-                <td ng-switch on="row.status">
+                <td ng-switch on="row.status" class="centerThis">
```
Comment (stackus) [inline public/templates/partials/common/problem_alerts.html]: Use the bootstrap `text-center` alignment class.

http://getbootstrap.com/css/#type-alignment

Comment (tangerine007) [inline public/templates/partials/common/problem_alerts.html]: I will center this bell if it is the last thing I do.

**Pull request merged**